docs/conf.py

```diff
@@ -0,0 +1,64 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("."))
+sys.path.insert(0, os.path.abspath("../"))
+sys.path.insert(0, os.path.abspath("../../"))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "Spot ROS2"
+copyright = "2023, BDAI"
+author = "BDAI"
```

Comment:
MASKOR & Boston Dynamics AI Institute

Reply:
Actually, @jbarry-bdai , do we want to include Oxford and BD? How does this work.

Reply:
We should include MASKOR and the Oxford Robotics Institute.  @METEORITENMAX @heuristicus is that the preferred name for your institutions for something like this?

I don't think we should include BD - they haven't asked us to or given us permission to.

Reply:
Thanks for the heads-up, I've put `MASKOR & Oxford Robotics Institute & Boston Dynamics AI Institute` for now. Please let me know if I need to make a change.